### PR TITLE
feat(dedicate.netapp): vrack services network tile on efs dashboard

### PR DESCRIPTION
**Pull request by qpavy**
ref: MANAGER-12540

| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/efs-vrack-services`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-12540
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- ~~[ ] Breaking change is mentioned in relevant commits~~

### CODE REVIEW

File: packages/manager/modules/netapp/src/dashboard/service.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    return this.Poller.poll(
+      `/storage/netapp/${storage.name}/network`,
+      {},
+      {
+        namespace: `network_${storage.name}`,
+        interval: 5000,
+        method: 'get',
+        successRule: (data) => {
+          return data.status !== NETWORK_STATUS.TO_CONFIGURE;
+        },
```
Comment (JacquesLarique) [inline packages/manager/modules/netapp/src/dashboard/service.js:65]:
```suggestion
        successRule: (data) => data.status !== NETWORK_STATUS.TO_CONFIGURE,
```

File: packages/manager/modules/netapp/src/dashboard/service.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    return { attachedSubnet, attachedEndpoint };
+  }
+
+  startNetworkPolling(storage) {
+    return this.Poller.poll(
+      `/storage/netapp/${storage.name}/network`,
+      {},
+      {
+        namespace: `network_${storage.name}`,
+        interval: 5000,
```
Comment (JacquesLarique) [inline packages/manager/modules/netapp/src/dashboard/service.js:61]: Could be a constant, not sure what we think about "magic numbers"

File: packages/manager/modules/netapp/src/dashboard/index/translations/Messages_fr_FR.json [hunk 3]
```diff
@@ -9,7 +9,13 @@
   "netapp_dashboard_network_configuration": "Configuration du réseau",
   "netapp_dashboard_network_configuration_status": "Statut",
   "netapp_dashboard_network_configuration_status_to_configure": "Prêt à configurer",
+  "netapp_dashboard_network_configuration_status_associating": "En cours de déploiement",
+  "netapp_dashboard_network_configuration_status_dissociating": "TODO",
```
Comment (darsene) [inline packages/manager/modules/netapp/src/dashboard/index/translations/Messages_fr_FR.json:13]: no french content ?

**Review by JacquesLarique — APPROVED** [review]

File: packages/manager/modules/netapp/src/dashboard/index/translations/Messages_fr_FR.json (hunk 3 above)
Comment (qpavy) [reply packages/manager/modules/netapp/src/dashboard/index/translations/Messages_fr_FR.json:13]: I am waiting for it but I will add it in my last PR for this project

**Review by antonymarion — APPROVED** [review]

**Review by darsene — APPROVED** [review]

**Pull request merged**